domain-services/ifix-master-data-service/src/main/java/org/egov/web/controllers/EatApiController.java

```diff
@@ -17,25 +18,33 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.validation.Valid;
 import java.io.IOException;
+import java.util.List;
 
 @javax.annotation.Generated(value = "org.egov.codegen.SpringBootCodegen", date = "2021-08-02T16:24:12.742+05:30")
 
 @Controller
-@RequestMapping("/eGovTrial/iFIX-Master-Data/1.0.0")
+@RequestMapping("/eat/v1")
 public class EatApiController {
 
     private final ObjectMapper objectMapper;
 
     private final HttpServletRequest request;
 
+    @Autowired
+    EATService eatService;
+
+    @Autowired
+    private ResponseHeaderCreator responseHeaderCreator;
+
     @Autowired
     public EatApiController(ObjectMapper objectMapper, HttpServletRequest request) {
         this.objectMapper = objectMapper;
         this.request = request;
     }
 
-    @RequestMapping(value = "/eat/v1/_create", method = RequestMethod.POST)
-    public ResponseEntity<EatResponse> eatV1CreatePost(@ApiParam(value = "Details for the new department + RequestHeader (meta data of the API).", required = true) @Valid @RequestBody EatRequest body) {
+    @RequestMapping(value = "/_create", method = RequestMethod.POST)
+    public ResponseEntity<EatResponse> eatV1CreatePost(@ApiParam(value = "Details for the new department + RequestHeader" +
+            " (meta data of the API).", required = true) @Valid @RequestBody EatRequest body) {
         String accept = request.getHeader("Accept");
```

Comment:
Remove this code and only keep the last line in the method i.e. return response code NOT_IMPLEMENTED.

Reply:
Done